tools/tcoffee/tcoffee.xml

```diff
@@ -0,0 +1,174 @@
+<tool id="t_coffee" name="T-Coffee" version="0.1">
```

Comment:
Can you please add ` profile="16.04"` to the tool tag?
This will hopefully fix the stderr error.